Examples/SnapinsSDKKotlinExample/README.md

```diff
@@ -21,9 +21,13 @@ Update the required string configurations in [strings.xml](app/src/main/res/valu
 valid data to ensure the Snap-ins SDK will function as expected.
 
 
-### Using Authenticated Users
+### Using Authentication
+
+In order to use authentication, developers can:
+* Provide a valid domain in [servers.xml](app/src/main/res/xml/servers.xml).
+* Use the provided default <a href="https://login.salesforce.com">login.salesforce.com</a> domain 
```

Comment:
Wonder if you should make make it clear that this is an either/or list of things... Maybe just say: "developer can either:" before the items...?

Reply:
Sure, works for me.